src/foundry/client/pixi/workers/texture-worker.d.ts

```diff
@@ -0,0 +1,125 @@
+export {};
+
+declare global {
+  /**
+   * Wrapper for a web worker meant to convert a pixel buffer to the specified image format
+   * and quality and return a base64 image
+   */
+  class TextureCompressor extends AsyncWorker {
+    /**
+     * @param name   - The worker name to be initialized
+     *                 (default: "Texture Compressor")
```

Comment:
Backticks are the convention with defaults.

Reply:
This is to make them inline code in markdown docs. For literals we still add what's needed for the literals. So in the end, it should look like this:
```
(default: `"Texture Compressor"`)
```